DocumentDataAPI/DocumentDataAPI/Data/Mappers/WordRatioMap.cs

```diff
@@ -5,12 +5,13 @@ namespace DocumentDataAPI.Data.Mappers;
 
 public class WordRatioMap : EntityMap<WordRatioModel>
 {
-    public const string DocumentId = "documents_id";
-    public const string Word = "word";
     public const string Amount = "amount";
+    public const string ClusteringScore = "clustering_score";
+    public const string DocumentId = "documents_id";
     public const string Percent = "percent";
     public const string Rank = "rank";
-    public const string ClusteringScore = "clustering_score";
+    public const string Tf_Idf = "tf-idf";
```

Comment:
```suggestion
    public const string TfIdf = "tf-idf";
```
C# bruger ikke snake case